operator/api/redpanda/v1alpha2/redpanda_clusterspec_types.go

```diff
@@ -256,16 +256,21 @@ type RedpandaConsole struct {
 	// Specifies whether a Deployment should be created for Redpanda Console.
 	// +kubebuilder:pruning:PreserveUnknownFields
 	Deployment *runtime.RawExtension `json:"deployment,omitempty"`
-	// Configures custom settings for Redpanda Console.
+	// Deprecated: Use `config` instead
 	// +kubebuilder:pruning:PreserveUnknownFields
 	Console *runtime.RawExtension `json:"console,omitempty"`
+	// Configures custom settings for Redpanda Console.
+	// +kubebuilder:pruning:PreserveUnknownFields
+	Config *runtime.RawExtension `json:"config,omitempty"`
 	// Configures console's Deployment's update strategy.
 	// +kubebuilder:pruning:PreserveUnknownFields
 	Strategy *runtime.RawExtension `json:"strategy,omitempty"`
-	// Settings for license key, as an alternative to secret.enterprise when a
-	// license secret is available
+	// Deprecated: Use `licenseSecretRef` instead.
```

Comment:
Can you denote the version of the console chart that the modified fields correspond to? When we get around to actually handling the migration path it'll be much more useful to see "this field is only read in V2" rather than "this field is deprecated".

Reply:
I do my best to create comment that is publicly available, but indicates the version of Console helm chart.